hedera-node/src/main/java/com/hedera/services/grpc/marshalling/ImpliedTransfers.java

```diff
@@ -41,38 +45,54 @@
 public class ImpliedTransfers {
 	private final ImpliedTransfersMeta meta;
 	private final List<BalanceChange> changes;
+	private final List<Pair<EntityId, List<CustomFee>>> entityCustomFees;
+	private final List<CustomFeesBalanceChange> customFeesBalanceChanges;
 
-	private ImpliedTransfers(ImpliedTransfersMeta meta, List<BalanceChange> changes) {
+	private ImpliedTransfers(ImpliedTransfersMeta meta, List<BalanceChange> changes,
+			List<Pair<EntityId, List<CustomFee>>> entityCustomFees,
+			List<CustomFeesBalanceChange> customFeeBalanceChanges) {
```

Comment:
Related to my comment on the protobuf [here](https://github.com/hashgraph/hedera-protobufs/pull/42/files#r657356764), imo we should rename `CustomFeesBalanceChange` to `AssessedCustomFee`.

So `ImpliedTransfers` would have a `List<AssessedCustomFee> assessedCustomFees` field that gives all the custom fees that must be paid for the `CryptoTransfer` to succeed.

Reply:
Renamed to `AssessedCustomFee` . Thanks @tinker-michaelj